app/templates/macros/toolkit_forms.html

```diff
@@ -1,9 +1,9 @@
-{% macro text(question_content, service_data, errors, question_number=None, get_question=None) -%}
```

Comment:
Yeeeeeeah.... in python, a function defined with only positional parameters would only accept those parameters, you would have to explicitly add a `**kwargs` parameter to allow that, so the syntax diverges a bit here - still, if there's a reason for it, go right ahead. I was just making a comment in case you were doing it just for the hell of it.